code/api/src/seeders/2-products.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         slug: 'belt-for-women',
         description: 'A very nice belt for women.',
         type: params.product.types.accessory.id,
+        style: "Classy",
         gender: params.user.gender.female.id,
```

Comment:
We're not going to forget this, but just for the sake of documentation, we need more seeds so we can have a lot of data, and we also need more style types.